katsdpimager/preprocess.py

```diff
@@ -261,14 +261,8 @@ def close(self):
         if logger.isEnabledFor(logging.INFO):
             filesize = self._file.id.get_filesize()
             n_compressed = np.sum(self._length)
```

Comment:
Maybe you should rename the variable too for good measure.

Reply:
That refers to a different compression (merging visibilities with the same quantised coordinates). A side benefit of getting rid of the gzip compression is that it avoids confusion between the two.

Reply:
Great! I suspected something was up...